keyboards/rura66/readme.md

```diff
@@ -0,0 +1,22 @@
+# RURA66
+
+![RURA66](https://github.com/ShinoharaTa/keyboards/blob/master/RURA66/manual/images/IMG_0236.jpg)
+
+これは66キー備えた、シンプルな左右分割型キーボードです。あなたの肩への負担軽減をサポートします。
+
+This is a simple left / right split keyboard with 66 keys. Supports reducing the burden on your shoulders.
+
+* Keyboard Maintainer: [T.Shinohara](https://github.com/ShinoharaTa)
+* Hardware Supported:  The Shiro PCBs, ProMicro supported. Using Cherry MX Profile Switches.
+* Hardware Availability: [Booth (Japanese Site)](https://shino3.booth.pm/items/1444895)
+* 日本国内ではBoothで入手できます: [Booth (Japanese Site)](https://shino3.booth.pm/items/1444895)
+
+![RURA66-USE](https://github.com/ShinoharaTa/keyboards/blob/master/RURA66/manual/images/IMG_0211.jpg?raw=true)
+
+Make for this keyboard default keymap (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```